src/rules/no-unused-modules.ts

```diff
@@ -291,13 +291,6 @@ const determineUsage = () => {
   }
 }
 
-const getSrc = (src?: string[]) => {
-  if (src) {
-    return src
-  }
-  return [process.cwd()]
-}
-
```

Comment:
This function is redundant now that we no longer have to comply with ljharb's ridiculous backward compatibility. 🙄